lib/ex_csv/parser.ex

```diff
@@ -82,10 +82,14 @@ defmodule ExCsv.Parser do
   end
 
   # NEWLINE
+  defp build(<<rt,nl>> <> rest, [[current_field | previous_fields] | previous_rows], %{return: rt, newline: nl, quoting: false} = config) do
+    build_newline(rest, current_field, previous_fields, previous_rows, config)
+  end
+  defp build(<<char>> <> rest, [[current_field | previous_fields] | previous_rows], %{return: char, quoting: false} = config) do
```

Comment:
It looks like `\r` was used by older Macs and by some terminals.  I included it because it was mentioned in #6 (2 birds, 1 stone type deal).